node/src/server-modules/GlideJson.ts

```diff
@@ -164,17 +168,64 @@ export class GlideJson {
      * ```
      */
     static async get(
-        client: GlideClient | GlideClusterClient,
+        client: BaseClient,
         key: GlideString,
         options?: JsonGetOptions & DecoderOption,
-    ): Promise<ReturnTypeJson> {
+    ): Promise<ReturnTypeJson<GlideString>> {
         const args = ["JSON.GET", key];
 
         if (options) {
             const optionArgs = _jsonGetOptionsToArgs(options);
             args.push(...optionArgs);
         }
 
-        return _executeCommand<ReturnTypeJson>(client, args, options);
+        return _executeCommand<ReturnTypeJson<GlideString>>(
+            client,
+            args,
+            options,
+        );
+    }
+
+    /**
+     * Toggles a Boolean value stored at the specified `path` within the JSON document stored at `key`.
+     *
+     * @param client - The client to execute the command.
+     * @param key - The key of the JSON document.
+     * @param path - The JSONPath to specify.
```

Comment:
`path` is optional - put it in options.
Document how command behaves without `path` and test.